spec/features/index_haml_home_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/pages/home'
+
+RSpec.describe 'Home', type: :feature, js: true do
+  let!(:home_page) { Home.new }
```

Comment:
Why do you need to create it before test?
```suggestion
  let(:home_page) { Home.new }
```